### ISSUE TITLE
The defination of high prediction confidence?

### ISSUE THREAD

**Issue by Lvxiang713** [bowang-lab/scGPT#181]
It was with great interest that I read your recent publication in Nature Methods titled "scGPT: toward building a foundation model for single-cell multi-omics using generative AI" . I found your work both intriguing and highly significant. However, upon thorough reading, I came across a problems: As described in the 'Attention mask for generative pretraining' section of your work and Supplementary Fig.1, during scGPT generation, only a few gene expression values with high prediction confidence are generated in each iteration. However, I have not seen how to define this high prediction confidence in the article. Could you please answer my question?
![image](https://github.com/bowang-lab/scGPT/assets/119480930/8262c2ea-54e6-40af-861d-defc9e5cec06)

**Comment by subercui** [comment]
Hi, thank you for your interest and question. I also saw some related questions from others and I would love to provide further explanation for this. While honestly speaking, I personally have been busy working on several new projects recently for a while. Could you please give me a few days for this? I will try to provide detailed explanations paired with code examples soon

**Comment by Lvxiang713** [comment]
Thank you very much. I am looking forward to your reply

**Comment by yusri-dh** [comment]
Is there any update about the definition of prediction confidence?

**Comment by Lvxiang713** [comment]
> Is there any update about the definition of prediction confidence?

No, I am still confused of the  prediction confidence

**Comment by SabrinaRichter** [comment]
From checking the code, I think the whole cell-prompt generation part was taken out at some point before creation of this repo. In the paper they write it is part of the training procedure, but if you look in the train function, the model is only run once and also loss wise there is only once the option to add the GEP loss to the loss. But even in that case, I would be curious what you tried for the confidence @subercui :-)
